pkgs/development/interpreters/python/tests.nix

```diff
@@ -138,7 +138,9 @@ let
     # test-overrideScope = let
     #  myPackages = python.pkgs.overrideScope extension;
     # in assert myPackages.foobar == myPackages.numpy; myPackages.python.withPackages(ps: with ps; [ foobar ]);
-  } // lib.optionalAttrs (python ? pythonAttr) {
+    #
+    # Note: `pythonInterpreters.pypy39_prebuilt.pythonAttr` has `pythonAttr == null`.
+  } // lib.optionalAttrs (python ? pythonAttr && python.pythonAttr != null) {
```

Comment:
Not sure I understand. AFAIU the difference is in prebuilt vs non-prebuild attribute presence in `pkgs`:

```
nix-repl> pythonInterpreters.pypy39.pythonAttr
"pypy39"

nix-repl> pythonInterpreters.pypy39_prebuilt.pythonAttr
null
```

Reply:
Okay. So should we have a `is_prebuilt = python: ...` lambda? And then the comment explaining we cannot use pypy prebuilt...because for some reason we do not have that attribute set?

Reply:
Moved the condition out to `is_prebuilt` helper and added a comment around lambda and it's call site.